.github/workflows/autoupdate.yml

```diff
@@ -0,0 +1,51 @@
+name: Update
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
Will you add a manual trigger `on: workflow_dispatch` and with an input parameter to specify an project, it'll be easier to debug the workflow.

Reply:
Sure! We can do that.